src/main/java/de/tum/in/www1/artemis/repository/StudentParticipationRepository.java

```diff
@@ -487,6 +487,9 @@ default StudentParticipation findByIdWithSubmissionsElseThrow(long participation
     @Query("select count(sp) from StudentParticipation sp left join sp.exercise exercise where exercise.id = :#{#exerciseId} and sp.testRun = false group by exercise.id")
     Long countParticipationsIgnoreTestRunsByExerciseId(@Param("exerciseId") Long exerciseId);
 
+    @Query("select count(sp) from StudentParticipation sp left join sp.exercise exercise where exercise.id = :#{#exerciseId} and sp.testRun = true group by exercise.id")
+    Long countParticipationsOnlyTestRunsByExerciseId(@Param("exerciseId") Long exerciseId);
```

Comment:
It would be nice if we achieve a more consistent and more similar and cleaner formatting of our queries. So, try to use textblocks and maybe use uppercase for sql statements. :) 
```suggestion
    @Query("""
    SELECT COUNT(sp) from StudentParticipation sp LEFT JOIN sp.exercise exercise
    WHERE exercise.id = :#{#exerciseId} AND sp.testRun = TRUE
    GROUP BY exercise.id
        """)
    Long countParticipationsOnlyTestRunsByExerciseId(@Param("exerciseId") Long exerciseId);
```